### DISCUSSION TITLE
Keyboard shortcut to needed for the "invalidate/reconnect" button

### DISCUSSION THREAD

**Discussion by mistu4u** [dbeaver/dbeaver#26865] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
My company has setup a db connection policy due to which the connection is invalidated after every 10 minutes of inactivity. At that time, it becomes cumbersome to click the mouse button to "reconnect" to the db.

**Describe the solution you'd like**
A keyboard shortcut to reconnect.

**Describe alternatives you've considered**
NA

**Additional context**
NA

**Comment by ShadelessFox** [comment, marked as answer]
Hello @mistu4u,

Have you tried setting up a shortcut here in the preferences?

![image](https://github.com/dbeaver/dbeaver/assets/35821147/06059f62-f785-431a-be47-267d2f1cffc5)

**Comment by mistu4u** [comment]
This works. Thank you!
